src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -59,6 +59,22 @@ def register_options(cls, register):
             advanced=True,
             help="The maximum timeout (in seconds) that can be set on a test target.",
         )
+        register(
+            "--junit-xml-dir",
+            type=str,
+            metavar="<DIR>",
+            default=None,
+            advanced=True,
+            help="Specifying a directory causes junit xml results files to be emitted under "
+            "that dir for each test run.",
+        )
+        register(
+            "--junit-family",
+            type=str,
+            default="xunit2",
+            advanced=True,
+            help="JUnit XML format family",
```

Comment:
It'd be helpful to link to the docs: https://docs.pytest.org/en/latest/reference.html#confval-junit_family

```suggestion
            help="The format of the generated XML file. See https://docs.pytest.org/en/latest/reference.html#confval-junit_family.",
```